pkg/pillar/go.mod

```diff
@@ -30,7 +30,7 @@ require (
 	github.com/jaypipes/ghw v0.8.0
 	github.com/k8snetworkplumbingwg/network-attachment-definition-client v1.5.0
 	github.com/lf-edge/edge-containers v0.0.0-20240207093504-5dfda0619b80
-	github.com/lf-edge/eve-api/go v0.0.0-20250304184518-532c94981505
```

Comment:
@jsfakian eve-api was already bumped right? You should not bump it here.... if is still needed though, please, do it in a separate commit..... like this: https://github.com/lf-edge/eve/commit/cac6b864e05b3834edaa5543004c997be17de27d

Reply:
It is in a separate commit: https://github.com/lf-edge/eve/pull/4671/commits/0aabafe1a6adb74b83bbdd401c45fbf97c108fde.

Reply:
I meant a separated commit only to eve-api, keep it separated from other vendor packages....